packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -2649,14 +2651,10 @@ function mountDehydratedSuspenseComponent(
     // have timed out. In theory we could render it in this pass but it would have the
     // wrong priority associated with it and will prevent hydration of parent path.
     // Instead, we'll leave work left on it to render it in a separate commit.
-
-    // TODO This time should be the time at which the server rendered response that is
-    // a parent to this boundary was displayed. However, since we currently don't have
-    // a protocol to transfer that time, we'll just estimate it by using the current
-    // time. This will mean that Suspense timeouts are slightly shifted to later than
-    // they should be.
     // Schedule a normal pri update to render this content.
-    workInProgress.lanes = laneToLanes(DefaultHydrationLane);
+    workInProgress.lanes = laneToLanes(
+      enableHydrationLaneScheduling ? DefaultLane : DefaultHydrationLane,
```

Comment:
I use the same kill switch even though this is kind of the revert of using the hydration lane less.

Reply:
Will this lower the priority below other things hydrating (maybe that's the intent) or is there a timeout or something that forces it?

Reply:
The sync unification is not complete so the semantics there are kind of nonsensical atm in edge cases. 

In general in the model I don't think it should be possible to have other work on the default hydration lane already since that would've already been done if we're hydrating at lower priority or if there is some then that's because we're currently doing it.

The only thing is that it might be batched with other default lane work like updates which might delay the commit but that's also kind of the point that if that work interleaves with this work then we shouldn't have to rewind.